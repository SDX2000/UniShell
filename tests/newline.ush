echo "================"
echo $SCRIPT_NAME
echo "================"

echo a; echo b;
echo c; echo d