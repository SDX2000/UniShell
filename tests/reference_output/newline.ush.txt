================
newline.ush
================
a
b
c
d
